README.md

```diff
@@ -112,4 +112,83 @@ argocd login --insecure $ARGOCD_SERVER --username $ARGOCD_USER --password $ARGOC
 
 argocd app delete $app-bootstrap --grpc-web -y
 argocd app list --grpc-web
-```
\ No newline at end of file
+```
+
+
+## Tools
+
+### `app-config.yaml` template
+
+The `template_app_config_local.sh` is a tool to help in the generation of 
+ the `app-config.yaml` file. It contains several resources that can be either 
+ enabled or not depending on the provided information.
```

Comment:
This text is superfluous `It contains several resources that can be either enabled or not depending on the provided information.` It can be removed

Reply:
done